### PR TITLE
Nose cones

### PR DESCRIPTION
**Pull request by davesrocketshop**
Thank you for creating a pull request to contribute to FreeCAD-macros!
To integrate your macro please make sure the following steps are complete:

- [ ] Please check this box if you're not submitting a new macro.   
- [x] Are you submitting a new macro ?  
- [x] Have you followed the ['How to submit a macro'](../README.md#how-to-submit-a-macro) section of the README.md ?  
- [x] Your macro has a [Description](../README.md#macro-description) in its header.  
- [x] Your macro has a [CamelCase name](../README.md#camelcase-macro-name).  
- [x] Your macro is named [appropriately](../README.md#macro-name-specifics).  
- [x] Your macro contains a [Metadata section](../README.md#macro-metadata) that immediately follows the header description.  
- [x] Your macro is Python3/Qt5 compliant and tested on the latest FreeCAD stable and development releases.  
- [ ] You're including documentation on how your macro works (bonus: screenshots and/or video on the Wiki)  
- [x] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)  
- [x] Commit message is titled in the following way `[MacroName] Short description`.
- [ ] Optional, write or update the changelog in the macro, from latest to oldest.

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.19 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=34586).

---

### CODE REVIEW

File: ParametricObjectCreation/NoseCones.FCMacro [hunk 1]
```diff
@@ -0,0 +1,1264 @@
...
+__Date__ = '2021-01-29'
+__License__ = 'LGPL-2.0-or-later'
+__Web__ = 'https://en.wikipedia.org/wiki/Nose_cone_design'
+__Wiki__ = ''
+__Icon__ = 'NoseCones.png'
+__Help__ = ''
+__Status__ = ''
+__Requires__ = ''
+__Communication__ = 'https://github.com/FreeCAD/FreeCAD-macros/issues/'
+__Files__ = ''
```
Comment (galou) [inline ParametricObjectCreation/NoseCones.FCMacro:65]: Please add `NoseCones.png` to the list of files.

File: ParametricObjectCreation/NoseCones.FCMacro [hunk 2]
```diff
@@ -0,0 +1,1264 @@
...
+
+from PySide import QtGui, QtCore
+import FreeCAD
```
Comment (galou) [inline ParametricObjectCreation/NoseCones.FCMacro:68]: Please use
```python
import FreeCAD as app
import FreeCADGui as gui
```

and use them in the code for consistency with other macros.

File: ParametricObjectCreation/NoseCones.FCMacro [hunk 3]
```diff
@@ -0,0 +1,1264 @@
...
+__Web__ = 'https://en.wikipedia.org/wiki/Nose_cone_design'
+__Wiki__ = ''
+__Icon__ = 'NoseCones.png'
+__Help__ = ''
+__Status__ = ''
+__Requires__ = ''
+__Communication__ = 'https://github.com/FreeCAD/FreeCAD-macros/issues/'
+__Files__ = ''
+
+from PySide import QtGui, QtCore
```
Comment (galou) [inline ParametricObjectCreation/NoseCones.FCMacro:67]: Please add a comment: `  # FreeCAD's specific package!!`

**Review by galou — CHANGES_REQUESTED** [review]
Thanks for your contribution!

The only requested change is `__Files__`.

**Comment by davesrocketshop** [comment]
The functionality of this macro has been moved to the Rocket workbench which should be released shortly. As such, this macro is no longer required.

I am withdrawing the pull request.

**Pull request closed without merging**